llvm/lib/Target/AArch64/AArch64Processors.td

```diff
@@ -863,6 +889,8 @@ def : ProcessorModel<"cortex-a720", NeoverseN2Model, ProcessorFeatures.A720,
                      [TuneA720]>;
 def : ProcessorModel<"cortex-a720ae", NeoverseN2Model, ProcessorFeatures.A720AE,
                      [TuneA720AE]>;
+def : ProcessorModel<"cortex-a725", CortexA57Model, ProcessorFeatures.A725,
```

Comment:
NeoverseN2Model

Reply:
Thanks, change adjusted locally. Apologies, I accidentally closed this PR when I forced pushed a different change yesterday.